src/components/Link/Link.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withTranslation } from 'react-i18next';
+import { compose } from 'lodash/fp';
+import { ReactComponent as LinkIcon } from './Link.svg';
+
+import './Link.scss';
+
+const propTypes = {
+  href: PropTypes.string.isRequired,
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  children: null,
+};
+
+const Link = ({ href, children }) => {
+  return (
+    <a
+      className="wkp-link"
+      href={href}
+      rel="noopener noreferrer"
+      target="_blank"
+    >
+      <LinkIcon />
+      {children}
+    </a>
+  );
+};
+
+Link.propTypes = propTypes;
+Link.defaultProps = defaultProps;
+
+export default compose(withTranslation())(Link);
```

Comment:
You never use the `t` function so you don't need withTranslation.

In fucntionnal component you can use the useTranslation hook instead, it avoids the use of compose then you can return a memoized Component : export default React.memo(Link);

Reply:
I have completely removed `compose(withTranslation())` as we do not use translations in this component.